src/v2/Apps/getAppNovoRoutes.tsx

```diff
@@ -4,10 +4,12 @@ import { artistRoutes } from "v2/Apps/Artist/artistRoutes"
 import { artistSeriesRoutes } from "./ArtistSeries/artistSeriesRoutes"
 import { artistsRoutes } from "v2/Apps/Artists/artistsRoutes"
 import { artworkRoutes } from "v2/Apps/Artwork/artworkRoutes"
+import { auctionsRoutes } from "./Auctions/auctionsRoutes"
 import { collectRoutes } from "v2/Apps/Collect/collectRoutes"
 import { consignRoutes } from "v2/Apps/Consign/consignRoutes"
 import { conversationRoutes } from "v2/Apps/Conversation/conversationRoutes"
 import { debugRoutes } from "./Debug/debugRoutes"
+import { exampleRoutes } from "./Example/exampleRoutes"
```

Comment:
Good call, especially if this is used as the basis of new routes.

Reply:
If this truly is an example it shouldn't be deployed live in production. I also feel the same about the baseline page, it doesn't provide any real value in production either.

Reply:
I can follow up on that in a separate PR; we have an admin only flag I need to port over.